cmd/thanos/bucket.go

```diff
@@ -42,34 +41,28 @@ var (
 func registerBucket(m map[string]setupFunc, app *kingpin.Application, name string) {
 	cmd := app.Command(name, "inspect metric data in an object storage bucket")
 
-	gcsBucket := cmd.Flag("gcs-bucket", "Google Cloud Storage bucket name for stored blocks.").
-		PlaceHolder("<bucket>").String()
-
-	s3Config := s3.RegisterS3Params(cmd)
+	bucketConf := cmd.Flag("objstore.config", "The yaml format configuration of bucket for stored blocks.").
```

Comment:
Can we improve this description by saying `The object store configuration in yaml format.`